### PR TITLE
User settings

### PR DESCRIPTION
**Pull request by UnisLTU**
Dark mode, user settings, group chat settings, on key press enter send message

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -2,8 +2,19 @@ import { Route, Routes } from "react-router-dom";
...
 import ErrorPage from "./ErrorPage";
+import { useEffect } from "react";
 
 const App = () => {
+  useEffect(() => {
+    localStorage.theme === "dark" || !("theme" in localStorage)
+      ? document.documentElement.classList.add("dark")
+      : document.documentElement.classList.remove("dark");
+
+    localStorage.theme === "dark"
```
Comment (pavelVCS) [inline src/App.tsx]: can be removed or moved I guess

File: src/components/Chat/Modals/SettingsModal.tsx [hunk 2]
```diff
@@ -3,17 +3,48 @@ import { ModalsEnum } from "../../../pages/Chat";
...
+import ImageUpload from "../ImageUpload";
 
 const SettingsModal = () => {
-  const { setModal } = useContext(DataContext) as DataContextProps;
+  const { setModal, setIsDarkMode, isDarkMode } = useContext(
+    DataContext
+  ) as DataContextProps;
+
+  const handleClick = () => {
+    localStorage.theme === "dark" || !("theme" in localStorage)
```
Comment (pavelVCS) [inline src/components/Chat/Modals/SettingsModal.tsx]: not sure that this is required here

**Review by pavelVCS — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (UnisLTU) [reply src/App.tsx]: Updated

File: src/components/Chat/Modals/SettingsModal.tsx (hunk 2 above)
Comment (UnisLTU) [reply src/components/Chat/Modals/SettingsModal.tsx]: Updated

**Pull request merged**